editor.planx.uk/src/pages/FlowEditor/components/Team/TeamMembers.tsx

```diff
@@ -1,19 +1,158 @@
+import Avatar from "@mui/material/Avatar";
 import Box from "@mui/material/Box";
+import Chip from "@mui/material/Chip";
+import Container from "@mui/material/Container";
+import Divider from "@mui/material/Divider";
+import { styled } from "@mui/material/styles";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import Typography from "@mui/material/Typography";
 import { Role, User } from "@opensystemslab/planx-core/types";
 import React from "react";
+import EditorRow from "ui/editor/EditorRow";
+
+const StyledAvatar = styled(Avatar)(({ theme }) => ({
+  background: theme.palette.background.dark,
+  color: theme.palette.common.white,
+  fontSize: "1em",
+  fontWeight: "600",
+  marginRight: theme.spacing(1),
+}));
+
+const StyledTableRow = styled(TableRow)(({ theme }) => ({
+  "&:nth-child(even)": {
+    background: theme.palette.background.paper,
+  },
+}));
 
 export type TeamMember = Omit<User, "teams" | "isPlatformAdmin"> & {
   role: Role;
 };
 
+const roleLabels: Record<string, string> = {
+  platformAdmin: "Admin",
+  teamEditor: "Editor",
+};
```

Comment:
It's technically possible for the query to return `teamViewer` and I think we also want to account for that here - even though it's quite rare!

Reply:
Added, thanks!